Automattic/wp-calypso
# Remove discount timer experiment and associated code

Related to https://github.com/Automattic/dotcom-forge/issues/1938

## Proposed Changes

We've called the post purchase countdown timer experiment (21081-explat-experiment) for the control. This PR removes the treatment we created in https://github.com/Automattic/wp-calypso/pull/73402

## Testing Instructions



* Load this PR
* Purchase a Premium plan
* You should see an upsell to the business plan without the countdown timer.
* Go ahead a purchase the business plan to test for good measure.

## Pre-merge Checklist
